modules/kubernetes-addons/cloudnative-pg/values.yaml

```diff
@@ -0,0 +1,7 @@
+resources:
+  limits:
+    cpu: 100m
+    memory: 200Mi
+  requests:
+    cpu: 100m
+    memory: 100Mi
```

Comment:
Remove this file. Leave the defaults comes with the Helm Chart. You can pass advanced configuration from Data on EKS Blueprint